src/te/schedule/message_passing.cc

```diff
@@ -62,6 +101,16 @@ void PassDownDomain(const Stage& stage,
     return actx->Simplify(indexdiv(a + (b - 1), b));
   };
 
+  auto minimum_or_later  = [actx](PrimExpr a, PrimExpr b) {
```

Comment:
```suggestion
  auto minimum_or_later  = [actx](const PrimExpr& a, const PrimExpr& b) {
```
Also do this for the above funciton `ceil_div`

Reply:
Made the changes.